Sources/tart/Commands/SetCommand.swift

```diff
@@ -1,8 +1,8 @@
 import ArgumentParser
 import Foundation
 
-struct Set: AsyncParsableCommand {
-  static var configuration = CommandConfiguration(abstract: "Modify VM's configuration")
+struct SetCommand: AsyncParsableCommand {
```

Comment:
All the files in `Commands` folder doesn't have `Command` suffix. What's the reasoning for this particular rename?

Reply:
> What's the reasoning for this particular rename?

`Set` command collides with the [`Set`](https://developer.apple.com/documentation/swift/set) collection.

Reply:
I don't think it's in the scope of this PR. There is no usage of Set. IMO we shouild either rename all the commands or wait util there will be a collision.

Reply:
Reverted in https://github.com/cirruslabs/tart/pull/103/commits/d59f4ec06686fa55d0e3c5e151980e59535990e5.